ci/fast-flake-check.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+curSystem=$(nix eval --raw --impure --expr builtins.currentSystem)
+
+nixEvalJobsArgs=()
```

Comment:
**question:** Why does it have to be such a long and involved process to do such a popular and needed thing...

Reply:
If you find out let me know cuz I'm also curious!